classes/Product.php

```diff
@@ -193,7 +195,7 @@ class ProductCore extends ObjectModel
      *
      * @see ProductInterface
      */
-    public $redirect_type = '';
+    public $redirect_type = ProductInterface::REDIRECT_TYPE_NOT_FOUND;
```

Comment:
No im not sure :D but the empty string is equal to 404(at least its handled equally), so idk how to make it right :thinking:

Reply:
still relevant

Reply:
When you look a the front productController indeed the default behaviour is to match `REDIRECT_TYPE_NOT_FOUND` So I guess it would make sense to force a real default value instead of an empty one
Although if we do that we should probably update `toggleStatus` as well so that it correctky set the default value, and maybe introduce a new const `ProductInterface::DEFAULT_REDIRECT_TYPE` that would contain `REDIRECT_TYPE_NOT_FOUND` this would allow to have a single place to modify if we want to change this behaviour someday

Reply:
Then I'm Ok with this change 👍  thanks for the explanations